### ISSUE TITLE
ARM support

### ISSUE THREAD

**Issue by PiecePaperCode** [philhartung/aes67-monitor#3]
Cross referenced on an issue of the lib audify. your lib dosent work wirh ARM on an Raspberry pi. Can you specyfie it in your readme that its x64? Because node runs on ARM so it would be assumed that you would support arm aswell but thats not the case so a specification would be lovely. https://github.com/almoghamdani/audify/issues/9

**Comment by philhartung** [comment]
Just because there is no prebuild of Audify available for ARM doesn't mean it wont't compile on ARM. I already used Audify on the Raspberry Pi 4 running Raspbian. I'll compile it on Raspbian and update you on the steps to compile it.

**Comment by philhartung** [comment]
I tested it and it seems that Audify 1.5.3 won't build on ARM. As version 1.5.3 was just released 12 hours ago I changed the Audify version used by aes67-monitor to 1.5.2. Version 1.5.2 compiles without any problems on Raspbian 10/Raspberry Pi 4 with the following steps. First update the software repositories and install necessary packages:
```
sudo apt update
sudo apt upgrade
sudo apt install git nodejs npm
sudo apt install cmake make gcc libasound2-dev
```
After that we can install aes67-monitor:
```
git clone https://github.com/philhartung/aes67-monitor.git
cd aes67-monitor
npm install
```
